.travis.yml

```diff
@@ -11,6 +11,7 @@ env:
 
 before_install:
   - sudo sh setup.sh
+  - npm config set registry http://ci.strongloop.com:4873/
```

Comment:
I agree. Our release process is not the fastest. I do not see any other way to get pass this if a repo depends heavily on travis.